definitions/redux-form_v5.x.x/flow_>=v0.22.1/redux-form_v5.x.x.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import React from 'react';
+
+declare module 'redux-form' {
+  declare type InputProps = {
+    name: string,
+    value: string | number | boolean,
+    valid: boolean,
+    invalid: boolean,
+    dirty: boolean,
+    pristine: boolean,
+    active: boolean,
+    touched: boolean,
+    viisted: boolean,
+    autofiller: boolean
+  };
+  declare type FormProps = {
+    active: string,
+    asyncValidate: Function,
+    asyncValidating: string | boolean,
+    destroyForm: Function,
+    dirty: boolean,
+    error: String,
+    fields: { [fieldName: string]: InputProps },
+    handleSubmit: Function,
+    initializeForm: (data:Object) => any,
+    invalid: boolean,
+    pristine: boolean,
+    resetForm: Function,
+    formKey: string,
+    submitting: boolean,
+    submitFailed: boolean,
+    touch: (...fields: Array<string>) => void,
+    touchAll: () => void,
+    untouch: (...fields: Array<string>) => void,
+    untouchAll: () => void,
+    valid: boolean,
+    values: Object
+  };
+  declare type FormConfig = {
+    fields: Array<string>,
+    form: string,
+    alwaysAsyncValidate?: boolean,
+    asyncBlurFields?: Array<string>,
+    asyncValidate?: (values: Object, dispatch: Function, props: Object) => Promise<void>,
+    destroyOnUnmount?: boolean,
+    formKey?: string,
+    getFormState?: Function,
+    initialValues?: Object,
```

Comment:
Docs suggest this is an object with only string values. Could we type this as `{[field: string]: string}`?